lib/utilities/strings.rb

```diff
@@ -151,7 +151,50 @@ def self.only_integers?(string)
     !(string =~ /[^\d\s]/i) && !integers(string).empty?
   end
 
+  # Parse a scientificAuthorship field to extract author and year information.
+  #
+  # If the format matches ICZN, adds parentheses around author name (if detected)
+  # @param [String] authorship
+  # @return [Array] [author_name, year]
+  def self.parse_authorship(authorship)
```

Comment:
A reminder to consider dwc_agent gem, does it help here?  The more we can unify logic to external the better.